CONTRIBUTING.md

```diff
@@ -61,6 +61,8 @@ or
 npm run make:login
 ```
 
+> Note: You'll need to `oc login` and restart `backend` each day.
```

Comment:
nit: We have a `Makefile` that contains `oc login` already, would it make sense to include that as an "or". I know `oc login` does the same thing.

Reply:
I'm not exactly sure if there are different setup options for different purposes, so I was a little hesitant to make many changes. Maybe we can do a bigger overhaul with more consensus.

The way @DaoDaoNoCode had me set up this part was:

- login at https://console-openshift-console.apps.ui-shared-odh.dev.datahub.redhat.com/dashboards customadmins with the clusteradminuser1 creds
- click the "copy login commands" from the user dropdown menu
- login again
- display and copy the `oc login` command under "Log in with this token"
- run that command in the terminal

as he mentioned above, this also required .env.local to have `OC_PROJECT=opendatahub`

Reply:
I see. `make login` does the same as `oc login`, I'm concerned that the first-time users(who maybe are new to OpenShift) might not know this and get confused. Also, we just mentioned `make login` immediately before this change, so wanted to make it consistent.

Reply:
@DaoDaoNoCode - the `make login` command does seem easier, I can skip all the steps in the list above and just run the make login command with the following env settings:

```
OC_URL=https://api.ui-shared-odh.dev.datahub.redhat.com:6443
OC_USER=clusteradminuser1
OC_PASSWORD=thepassword-idk-if-i-should-post-the-real-thing:)
OC_PROJECT=opendatahub
```

should i update accordingly?

Reply:
the contributing readme already contains information about setting up .env.local, i'll just make a more generic "reauthenticate using one of the above steps" i think